### DISCUSSION TITLE
Styler as GitHub action

### DISCUSSION THREAD

**Discussion by elena-krismer** [jpquast/protti#211] [Ideas]
Hi, 
We could use  the R-package `styler`  (`styler::style_pkg`) with GitHub Actions to automatically tidy our code each time we create a PR.  https://usethis.r-lib.org/reference/tidyverse.html
Let me know if you think it's worth the integration, or if you would prefer to tidy your code manually. :) @jpquast

**Comment by jpquast** [comment]
That sounds great! I always did it manually but doing it automatically is certainly better. If you know how to add it easily to the workflow you can go ahead!

**Comment by jpquast** [comment]
We should still try to get it to trigger tests as well. I think this is a bit more difficult since there is some sort of protection from GitHub to not allow this since it can trigger infinite loops. But there is apparently some ways around it:
https://github.com/orgs/community/discussions/57484
